crates/torii/src/cli.rs

```diff
@@ -29,13 +35,79 @@ mod types;
 #[cfg(test)]
 mod tests;
 
+pub(crate) fn dojo_metadata_from_workspace(ws: &Workspace<'_>) -> Option<DojoMetadata> {
+    Some(ws.current_package().ok()?.manifest.metadata.dojo())
+}
+
+#[derive(Default, Deserialize, Debug, Clone)]
+pub(crate) struct DojoMetadata {
```

Comment:
This introduces duplication as now we have two definitions of `DojoMetadata`. For now, we can move this struct and its related stuff to the `dojo-world` crate under `metadata`  module or something.